Tasks/ServiceFabricDeploy/deploy.ps1

```diff
@@ -31,6 +31,7 @@ try {
     $compressPackage = [System.Boolean]::Parse((Get-VstsInput -Name compressPackage))
     $skipUpgrade =  [System.Boolean]::Parse((Get-VstsInput -Name skipUpgradeSameTypeAndVersion))
     $skipValidation =  [System.Boolean]::Parse((Get-VstsInput -Name skipPackageValidation))
+    $unregisterUnusedVersions = [System.Boolean]::Parse((Get-VstsInput -Name unregisterUnusedVersions))
```

Comment:
Remember to run `node make.js test --task ServiceFabricDeploy` to test the change. You'll need to add mocking of the new input to the existing tests.